panoptes/utils/images/fits.py

```diff
@@ -88,6 +90,29 @@ def get_solve_field(fname, replace=True, remove_extras=True, **kwargs):
     to complete, populates a dictonary with the EXIF informaiton and returns. This is often
     more useful than the raw `solve_field` function
 
+    Example:
+
+    >>> from panoptes.utils.images import fits as fits_utils
+
+    >>> # Get our fits filename
+    >>> fits_fn = getfixture('unsolved_fits_file')
+
+    >>> # Perform the Solve
+    >>> solve_info = fits_utils.get_solve_field(fits_fn)
+
+    >>> # Show solved Filename
+    >>> solve_info['solved_fits_file']
+    '/var/panoptes/panoptes-utils/panoptes/tests/data/unsolved.fits'
+
+    >>> #Try to pass a suggested location
+    >>> ra = 15.23
+    >>> dec = 90
```

Comment:
Might be good to get realistic coordinates. Or maybe show an example from `astropy`.